features/suzie_searches_by_free_text.feature

```diff
@@ -50,40 +47,36 @@ Feature: Suzy Singer searches by free text
     When I go to the home page
     And I fill in "search" with "overthrow the"
     And I press "Search"
-    But I should see the following search results table:
-      | Overthrow the Wombles View | 1          |
+    But I should see the following search results:
+      | Overthrow the Wombles | 1 signatures          |
```

Comment:
I'll add a story to clean up our "1 signatures" problem. It also exists in trending.

https://www.pivotaltracker.com/story/show/96644444